dotCMS/src/integration-test/java/com/dotcms/saml/IdentityProviderConfigurationFactoryTest.java

```diff
@@ -0,0 +1,46 @@
+package com.dotcms.saml;
+
+import com.dotcms.security.apps.AppsAPI;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.dotmarketing.util.Config;
+import org.apache.felix.framework.OSGIUtil;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.security.UnrecoverableKeyException;
+
+import static org.mockito.Matchers.anyCollection;
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class IdentityProviderConfigurationFactoryTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5956529)